src/libraries/System.Private.CoreLib/src/System/Threading/Tasks/Task.cs

```diff
@@ -2845,6 +2846,7 @@ private bool SpinThenBlockingWait(int millisecondsTimeout, CancellationToken can
                 try
                 {
                     AddCompletionAction(mres, addBeforeOthers: true);
+#pragma warning disable CA1416 // Validate platform compatibility, Need proper handling, affect all Wait(...) public APIs
```

Comment:
Should we annotate all Task.Wait(...) overloads unsupported on browser or create an issue? cc @marek-safar @stephentoub

Reply:
I don't think we want to, we need browser-specific implementation. Please create an issue and link if from here. I'm curious how this works today, I guess we are lucky and hit SpinWait all the time

@lewing

Reply:
Created https://github.com/dotnet/runtime/issues/44622

Reply:
Task.Wait() only works if the task is already completed, we only have one thread there is no way to sync Wait on a task and have it complete after the fact.